plugins/modules/win_net_adapter_feature.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, ライトウェルの人 <jiro.higuchi@shi-g.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENTATION = r'''
+---
+module: win_net_adapter_feature
+short_description: Enable or disable certain network adapters.
+description:
+     - Enable or disable ms_tcpip6 of a certain network adapter or all the network adapters.
+options:
+  interface:
+    description:
+      - Name of Network Adapter Interface. For example, 'Ethernet0' or '*'.
+    type: str
+    required: yes
+  state:
+    description:
+      - Specify the state of ms_tcpip6 of interfaces.
+    type: str
+    choices:
+      - enable
+      - disable
+    required: yes
+  componentID:
+    description:
+      - Specify the below componentID of network adapters.
+      - componentID (DisplayName)
+      - ms_implat (Microsoft Network Adapter Multiplexor Protocol)
+      - ms_lltdio (Link-Layer Topology Discovery Mapper I/O Driver)
+      - ms_tcpip6 (Internet Protocol Version 6 (TCP/IPv6))
+      - ms_tcpip (Internet Protocol Version 4 (TCP/IPv4))
+      - ms_lldp (Microsoft LLDP Protocol Driver)
+      - ms_rspndr (Link-Layer Topology Discovery Responder)
+      - ms_msclient (Client for Microsoft Networks)
+      - ms_pacer (QoS Packet Scheduler)
+      - If you'd like to set custom adapters like 'Juniper Network Service',
+      - confirm the componentID by 'Get-NetAdapterBinding' cmdlet.
+    type: str
+    required: yes
+
+author:
+  - ライトウェルの人 (@jirolin)
```

Comment:
There's no need to change this, it will be interesting to see if our build tools will work with non-ascii characters. It should be fine but merely just a comment that I haven't seen a module use Japanese characters before.

Reply:
As you say, it is the rare case that  a module use Japanese characters.
I would like to leave my account name as it is.

Reply:
That's perfectly fine, sorry I didn't mean to say you should change it. I was merely trying to say that I've never encountered it before and I hope the doc builder can handle it fine (which it should) :).